pontoon/machinery/views.py

```diff
@@ -9,7 +9,8 @@
 from django.http import HttpResponse, HttpResponseBadRequest, JsonResponse
 from django.shortcuts import get_object_or_404, render
 from django.utils.datastructures import MultiValueDictKeyError
-from suds.client import Client, WebFault
+from zeep import Client
+from zeep.exceptions import Fault as WebFault
```

Comment:
Decided to leave as WebFault as Fault is casual word and less meaningful. Should I leave it as Fault?